Automattic/jetpack
# Admin Page: Move to Core translation functions -- PART 2

#### Changes proposed in this Pull Request:

This is part 2, follow-up of #13527. We're moving away from `translate` and using the `@wordpress/i18n` package instead.

This PR focusses on updating all components used throughout the dashboard.

**Note: this PR is open against the branch used for part 1 in #13527 for now, hence the "Do not merge" label. Once the other PR is merged I'll update the base branch for this PR.**

#### Jetpack product discussion

* p1HpG7-99c-p2
* Primary issue: #16481 

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

Nothing should change at first sight.

* Everything should look good in the admin dashboard.
* Try activating and deactivating modules, changing settings, connecting / disconnecting
* Try using filters to trigger alternate views; something like #8934 for example.
* Try enabling development mode in different ways: `add_filter( 'jetpack_development_mode', '__return_true' );`, `define( 'JETPACK_DEV_DEBUG', true );`, both.

#### Proposed changelog entry for your changes:

* TBD
